src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -191,7 +191,7 @@ private static int FindAndParseNextNumber(string text, ref int pos)
             return num;
         }
 
-        private static int ProcessorCountCore => (int)Interop.Sys.SysConf(Interop.Sys.SysConfName._SC_NPROCESSORS_ONLN);
+        public static int ProcessorCount => (int)Interop.Sys.SysConf(Interop.Sys.SysConfName._SC_NPROCESSORS_ONLN);
```

Comment:
I spent quite some time tracking down an issue in Mono that was related to relying on _SC_NPROCESSORS_ONLN on ARM: https://github.com/mono/mono/pull/2733#issuecomment-193909102, might be good to keep this in mind.